### DISCUSSION TITLE
add build with sqlite_icu

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#4967] [Q&A] [answered]
Hi, can you add Pocketbase version with tags: -tags "sqlite_icu", for utf-8 support search ?

**Comment by ganigeorgiev** [comment, marked as answer]
No, I'm not planning adding it as it could cause confusion and further slow the build release time.

Users can add it on their own if it is needed.

**Reply by ganigeorgiev** [reply]
Also note that the icu build tag is available only when using the CGO driver which means that the binaries will not be statically linked out of the box and may not work on every environment.
